ui/components/multichain/multichain-token-list/multichain-token-list.scss

```diff
@@ -0,0 +1,15 @@
+.multichain-token-list {
+  &__container-cell {
+    cursor: pointer;
+
+    &:hover,
+    &:focus-within {
+      background-color: var(--color-background-default-hover);
+    }
+
+    &--text-container {
+      width: 100%;
+      cursor: pointer;
```

Comment:
Isn't the entire component an `<a>` tag so the cursor should already be a pointer?
```suggestion
```